02.calendar/cal.rb

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env ruby
+
+require 'optparse'
+require 'date'
+
+params = ARGV.getopts("", "y:#{Date.today.year}", "m:#{Date.today.month}")
+month = params["m"].to_i
+year = params["y"].to_i
+
+start_date = Date.new(year,month,1) 
+end_date = Date.new(year,month,-1)  
+day_of_week = start_date.wday
+
+wdays = ["日 " + "月 " + "火 " + "水 " + "木 " + "金 " + "土 "]
+
+  puts "#{month}月 #{year}".center(20)
+  puts wdays
```

Comment:
```suggestion
  puts "日 月 ..."
```

のように直接文字列を出力した方がシンプルです

Reply:
puts "日 月 火 水 木 金 土"の書き方に変えました！